x-pack/test_serverless/functional/test_suites/observability/config.feature_flags.ts

```diff
@@ -21,16 +21,11 @@ export default createTestConfig({
   kbnServerArgs: [
     '--xpack.infra.enabled=true',
     '--xpack.infra.featureFlags.customThresholdAlertsEnabled=true',
-    '--xpack.security.roleManagementEnabled=true',
-    `--xpack.cloud.serverless.project_id='fakeprojectid'`,
-    `--xpack.cloud.base_url='https://cloud.elastic.co'`,
-    `--xpack.cloud.organization_url='/account/members'`,
-    `--xpack.spaces.maxSpaces=100`, // enables spaces UI capabilities
   ],
   // load tests in the index file
   testFiles: [require.resolve('./index.feature_flags.ts')],
 
   // include settings from project controller
   // https://github.com/elastic/project-controller/blob/main/internal/project/observability/config/elasticsearch.yml
-  esServerArgs: ['xpack.ml.dfa.enabled=false', 'xpack.security.authc.native_roles.enabled=true'],
+  esServerArgs: ['xpack.ml.dfa.enabled=false'],
```

Comment:
__question:__ When the O11y solution is ready to start working on custom roles functionality and would like to run tests related to roles regularly - locally, on CI, and in MKI - but aren’t ready to enable roles in production yet, what would be our recommendation? Will they need to remove "skip" flags and add the necessary overrides in GitOps to enable custom roles functionality in QA only to ensure we can run MKI tests?

Reply:
Good question. I guess we have a couple of options:
1. Retain the feature flag tests for OBLT to use (however, I don't think these get run on MKI)
2. Adjust skip flags and FTR config, enable in QA/Staging, keep disabled in prod

Maybe @pheyos has a better suggestion?

Reply:
I think we only rarely want to enable feature flags broadly in a whole env for testing and custom roles for Observability does not really qualify for an exception there IMO. Here's what I think they should do once they're ready to test:

* get feature flag tests in - doesn't run on MKI but shows that things are functional
* create one or more test projects in MKI and override the setting for these projects in order to tests end-to-end
  * run the feature flag tests against this project

If that's all looking good:
* make sure no existing tests are breaking if custom roles are enabled
* step by step enabled the feature in QA, Staging and Production
* then move the feature flag tests to regular tests

Reply:
Thanks, Robert! @azasypkin Do you think we should we split out the roles tests and set the feature flag test back up for OBLT now? Not sure what their timeframe is for enabling custom roles.

Reply:
Thanks, @jeramysoucy. TL;DR: let’s keep what you have here.

Since there are no immediate signs that O11y is planning to introduce custom roles anytime soon (though I believe this might change quickly with GA), I don’t think we need to make the testing setup more complex than it needs to be. Once the O11y team is ready, we’ll propose what Robert outlined above and provide any necessary assistance if needed.